platform/server/http/__tests__/HttpServer.test.ts

```diff
@@ -12,8 +14,17 @@ let app: any;
 let port: number;
 
 beforeEach(() => {
+  const log: Logger = {
```

Comment:
Not something we need to fix in this PR: There is a logger mock in `../../../logging/__mocks__`, not sure if that works out-of-the-box here. We need some way of sharing this across tests, though, just so we don't have to keep mocks like this up-to-date. Jest has some automatic mocking support, but I haven't looked at it yet.

Reply:
Agree